notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -0,0 +1,296 @@
+    "from azureml.core.runconfig import RunConfiguration\n",
+    "from azureml.contrib.notebook.notebook_run_config import NotebookRunConfig, PapermillExecutionHandler\n",
+    "from azureml.core.runconfig import DEFAULT_CPU_IMAGE\n",
+    "from azureml.core.conda_dependencies import CondaDependencies\n",
+    "\n",
+    "print(\"This notebook was created using version 1.0.21 of the Azure ML SDK\")\n",
+    "print(\"You are currently using version\", azureml.core.VERSION, \"of the Azure ML SDK\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Connect to an AzureML workspace\n",
+    "\n",
+    "An [AzureML Workspace](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.workspace.workspace?view=azure-ml-py) is an Azure resource that organizes and coordinates the actions of many other Azure resources to assist in executing and sharing machine learning workflows. In particular, an Azure ML Workspace coordinates storage, databases, and compute resources providing added functionality for machine learning experimentation, deployment, inferencing, and the monitoring of deployed models.\n",
+    "\n",
+    "**If you are trying to create or connect to a workspace for the first time in this environment, uncomment and run the below cell. ** It will load code necessary to create workspace. After code is loaded, run the cell again to create the workspace. It will create `aml_config\\config.json` file which contains workspace configuration for future notebook runs. See [this tutorial](https://docs.microsoft.com/en-us/azure/machine-learning/service/setup-create-workspace#portal) to locate information such as subscription id."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Wrote the config file config.json to: /data/home/testuser/notebooks/Recommenders/notebooks/00_quick_start/aml_config/config.json\n",
+      "AzureML workspace name:  setup-ws\n"
+     ]
+    }
+   ],
+   "source": [
+    "# %load ../../scripts/create_workspace.py"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Found the config file in: /data/home/testuser/notebooks/Recommenders/notebooks/00_quick_start/aml_config/config.json\n"
+     ]
+    }
+   ],
+   "source": [
+    "ws = Workspace.from_config()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example uses run-based AzureML Compute, which requires minimum setup. Alternatively, you can use persistent compute target (cpu or gpu cluster) or remote virtual machines. For more details, see [How to set up training targets](https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-set-up-training-targets).\n",
```

Comment:
I think all of these are super useful for Azure users of the repo, but again, do we want to put it somewhere in general, or reference to an existing article?

Reply:
See reply above